package.json

```
@@ -14,9 +14,11 @@
     "postversion": "git push && git push --tags",
     "start": "node dist/index.js",
     "test": "dotenv -e env.example -- jest",
-    "lint": "tslint --project ."
+    "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx"
```

you could add a fix linting script
```suggestion
    "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
    "lint:fix": "eslint . -c .eslintrc.js --ext .ts,.tsx --fix"
```

---

I talked with @balanza about that, we think that "yarn lint --fix" is less verbose than the "yarn lint-autofix" that I've seen somewhere else. Even if "yarn lint:fix" is effectively less verbose than the others, I think that it's more natural to add the "--fix" when launching yarn lint because the command itself suggests this. What do you think?

---

Nice idea. We can keep the standard `yarn lint --fix`. The custom script inside the package json is useful only as a suggestion for contributors.

---

Contributors can see the lint message that says to ass --fix too. Let's keep it simple.